### DISCUSSION TITLE
Actividad 0 - Muñoz, Dalma

### DISCUSSION THREAD

**Discussion by dalmamunoz** [frlp-utn-ingsoft/catedra#54] [Show and tell]
https://github.com/dalmamunoz/ingsoftware

Al final si necesite crear un entorno virtual para evitar conflictos.
#Windows 
`py -m venv .env
`
`.env\scripts\activate
`
`deactivate`

Gracias

**Comment by RodrigoJacznik** [comment]
Hola,
buenos commits, la app funciona bien (muy bien agregando una encuesta extra 👍 ).
Actividad aprobada
